tailwind.config.cjs

```diff
@@ -58,6 +58,7 @@ module.exports = {
     },
     zIndex: {
       'header': '2',
+      'popup': '3',
```

Comment:
I would expect that 1 and 2 will be enough :D

Reply:
Done.